app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -3,6 +3,8 @@
 
 module Foreman::Model
   class Ovirt < ComputeResource
+    using Foreman::HttpProxy::RestClientRefinements
```

Comment:
I'm not too familiar with refinements, but why is it needed here? You don't pass any credentials to RestClient.

Reply:
Eeh, neither am I to be honest. It is not strictly needed, but will anyone remember to add it in case we decide to make ovirt CR use a proxy?

Reply:
Fair enough.